x-pack/plugins/integration_assistant/server/graphs/ecs/pipeline.ts

```diff
@@ -4,7 +4,7 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-/* eslint-disable @typescript-eslint/no-explicit-any */
+
```

Comment:
This file is indeed owned by @elastic/security-scalability .

Unfortunately, it makes no sense to accept your suggestion as this rules has been disabled for a reason. We get three warnings if we disable the rule for this file:

<img width="676" alt="image" src="https://github.com/user-attachments/assets/fe88b930-e79e-416c-b62f-6fb6da6e9b43">

Let's revert this:

```suggestion
/* eslint-disable @typescript-eslint/no-explicit-any */
```

Reply:
Thanks @ilyannn. Looks like change was introduced in the [https://github.com/elastic/kibana/commit/82f16e79759bfa4951e3fcd08e6f2c865178a56c#diff-9c3801d20ff12735c7d0433340[…]ef83f0215a8f8dd5154e6880159f6aL8](https://github.com/elastic/kibana/commit/82f16e79759bfa4951e3fcd08e6f2c865178a56c#diff-9c3801d20ff12735c7d0433340ac2b3b9aef83f0215a8f8dd5154e6880159f6aL8). Reverted now.